test/tasks_screen/models/task_database_operations_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:lachies_life_planner/shared/global_config.dart';
+import 'package:lachies_life_planner/tasks_screen/models/task-database-operations.dart';
+import 'package:lachies_life_planner/tasks_screen/models/task.dart';
+
+import '../../utils/mock_firestore_data.dart';
+
+void main() {
+  group('Task', () {
+    MockFirestoreInstance mockFirestoreInstance;
+
+    setUp(() {
+      mockFirestoreInstance = MockFirestoreInstance();
+      setFirestoreInstance(mockFirestoreInstance);
+    });
+
+    group('addTask()', () {
+      test('adds task to Firestore', () async {
+        addTask(mockTask);
+
+        DocumentSnapshot dbData = await mockFirestoreInstance.collection('tasks').doc(mockTask.id).get();
+        expect(dbData.data(), mockTask.toJson());
+      });
+
+      test('can be used multiple times', () async {
+        for (Task task in mockTaskList) {
+          addTask(task);
+
+          DocumentSnapshot dbData = await mockFirestoreInstance.collection('tasks').doc(task.id).get();
+          expect(dbData.data(), task.toJson());
```

Comment:
Can make this into a method, since it will be used multiple times during testing.  Maybe even in the app.  There may be a case where we want to add all tasks in a list

Reply:
Done